src/NUnitConsole/nunit3-console.tests/ResultReporterTests.cs

```diff
@@ -169,11 +177,23 @@ public void TestsNotRunTest()
             Assert.That(report, Is.EqualTo(expected));
         }
 
-        #region Helper Methods
+        [Test]
+        public void TestParameterSettingsWrittenCorrectly()
+        {
+            var expected = new[] {
+                "    TestParameters: 1=d;2=c",
+                "    TestParametersDictionary:",
+                "        1 -> d",
+                "        2 -> c",
+                "    NumberOfTestWorkers: 8"
+            };
 
-        private TestEngineResult AddMetadata(TestEngineResult input)
+            var report = GetReportLines(_reporter.WriteRunSettingsReport);
+            Assert.That(report, Is.SupersetOf(expected));
+        }
+
```

Comment:
Tests both forms of the `TestParameters` setting, to ensure ongoing support, until such a time as we chose not to. `Is.SubsetOf` is an attempt to make the test more maintainable than expecting an exact string.

Reply:
👍 (but you probably meant `Is.SupersetOf` and not `Is.SubsetOf` 😄)

Reply:
I think what I have is correct? I'm expecting the actual report to contain more stuff than the `expected` array. Or am I missing something? 🙂

Reply:
OH - I explained it wrong in my line comment! Apologies! 😅